lib/id3c/labelmaker.py

```diff
@@ -208,10 +208,28 @@ class CollectionsScanKiosksLayout(LabelLayout):
 
 
 class CollectionsCliaComplianceLayout(LabelLayout):
```

Comment:
If we go more generic we would have a second layout class for Clia.

Reply:
How were you imagining invoking a second layout class?

Reply:
I was thinking we would pass in for the --layout option the name of the layout class itself.

Reply:
I have some reservations about evaluating a string literally to instantiate a python class. I would personally prefer to create multiple, callable layouts within a barcode class (e.g. small, compact, or minified) as suggested by @kschwabefry in a different thread in this PR.

Reply:
Going with the --layout option is good.